code/modules/modular_armor/modular.dm

```diff
@@ -202,6 +202,9 @@
 	if(.)
 		return
 
+	if(user.action_busy)
+		return FALSE
```

Comment:
add this to each?

Reply:
Each what

Reply:
each `x_act`, since the issue will happen with each of them no?